### ISSUE TITLE
[Question]

### ISSUE THREAD

**Issue by KIKO-9** [materialsvirtuallab/matpes#14]
Dear MatPES team,

In the paper "[A Foundational Potential Energy Surface Dataset for Materials](https://arxiv.org/pdf/2503.04070)" , I noticed that the CHGNet model trained on MPTrj in **Table 1** seems to show significant overfitting on the test set, which is different from the [CHGNet-v0.3.0](https://github.com/CederGroupHub/chgnet/tree/main/chgnet/pretrained/0.3.0). Was the test set randomly selected from MPTrj?

Also, I have an unrelated question: Can a model trained on MPTrj be directly used to make predictions on MatPES-PBE samples?

Thanks!

**Comment by shyuep** [comment]
@bowen-bd I will let you answer the CHGNet question.

As for the second question, the answer is yes in certain system where the functional is PBE nad the PSP is similar. We are using PBE64 from VASP while MPTrj is using a very old PBE52, but once you correct for the atomic energies, it shouldn't be that different. But MPTrj also uses PBE+U for some TM systems. Those cannot be compared directly to MatPES PBE only calculations. In general, I would suggest deprecating all MPTrj based models. That dataset is really not PES quality (I am saying this as someone who came up with the first version of it).

**Comment by KIKO-9** [comment]
Ah, I get it! Thank you for your detailed reply!

**Comment by bowen-bd** [comment]
Hi @KIKO-9 , for table 1, the train and val errors of MPtrj CHGNet are measured on the MPtrj dataset, but the test error is measured on the subset of MatPES. And the MPtrj-CHGNet checkpoint comes the [MatGL-version CHGNet](https://github.com/materialsvirtuallab/matgl/tree/main/pretrained_models/CHGNet-MPtrj-2023.12.1-2.7M-PES), not the one from CederGroupHub that you pointed out. 
The high test error of MPtrj-trained CHGNet on MatPES (698 meV/atom) is due to the inclusion of PBE+U datapoints and [MP20](https://github.com/materialsproject/pymatgen/blob/v2025.6.14/src/pymatgen/entries/compatibility.py#L948-L1265) compatibility in the MPtrj dataset, which made MPtrj training labels not directly compatible with MatPES PBE labels.
We have mentioned this in the manuscript: 
> Additionally, the exceptionally high energy error in the MPtrj CHGNet model stems from the inclusion of both PBE and PBE+U calculations in its training data.

In fact, not only MPtrj CHGNet, the MPF and OMat24 models also experienced very high test errors on MatPES, this is due to label incompatibility.
